test/functional/p2p_node_network_limited.py

```
@@ -80,11 +80,18 @@ def run_test(self):
+        with self.nodes[0].wait_for_debug_log(
+                [b"Ignore block request below NODE_NETWORK_LIMITED threshold"], timeout=5):
+            try:
+                self.connect_nodes(0, 2)
+            except AssertionError as e:
+                if "peer disconnected" not in str(e):
+                    raise
+                self.log.info("NODE_NETWORK_LIMITED peer disconnected unsynced peer before connect completed")
+            try:
+                self.sync_blocks([self.nodes[0], self.nodes[2]], timeout=5)
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)